crates/ty_server/src/document/notebook.rs

```diff
@@ -1,10 +1,9 @@
+use crate::{PositionEncoding, TextDocument};
```

Comment:
super nit: I don't know how strict we are with our `use` sections, but I don't think we want `crate` imports in the same section as external-but-not-stdlib imports. (If anything, the `crate` and `super` sections below should be merged)